libs/sdk-ui-dashboard/src/dashboard/Dashboard.tsx

```diff
@@ -140,6 +140,26 @@ export interface IDashboardProps {
      */
     dashboardRef: ObjRef;
 
+    /**
+     * Data product identifier.
+     * This property is required, if the current backend implementation allows usage of data product identifier
+     * and data product identifier is set (workspace is provisioned via LCM).
+     *
+     * It's used to resolve drill to custom url data product identifier parameter.
+     * For more details, see: https://help.gooddata.com/doc/enterprise/en/dashboards-and-insights/kpi-dashboards/drilling-in-kpi-dashboards/drill-to-url-from-kpi-dashboards#DrilltoURLfromKPIDashboards-CustomURL
```

Comment:
Use the Share version of the link to avoid dead links if the name of the article changes. Also please use the `@link` TSDoc feature fo once we add this package to the apidocs.

Same for the other prop